release_notes/release_notes.stex

```diff
@@ -116,6 +116,14 @@ Online versions of both books can be found at
 %-----------------------------------------------------------------------------
 \section{Functionality Changes}\label{section:functionality}
 
+\subsection{Simpified lookup of property values (10.2.0)}
```

Comment:
\subsection{Added \scheme{property-value} to look up compile-time property values (10.2.0)}